MANIFEST.in

```diff
@@ -1,6 +1,2 @@
-include pyhf/schemas/1.0.0/defs.json
-include pyhf/schemas/1.0.0/measurement.json
-include pyhf/schemas/1.0.0/model.json
-include pyhf/schemas/1.0.0/workspace.json
-include pyhf/schemas/HistFactorySchema.dtd
+graft src
```

Comment:
@kratsg It would be nice to be able to [use graft to be super explicit on what we're packaging](https://blog.ionelmc.ro/2014/06/25/python-packaging-pitfalls/). I'm worried that I've messed up the schema deployment though. Can you explain a bit more about what how the paths are setup for the deployment? Is my updating the symlink like I've done in this PR enough?

Reply:
`graft src` and `include pyhf/schemas` are incompatible. You do not include pyhf source code in the `MANIFEST.in` which is what you've done. You only want to include the schemas as part of _additional_ files to include.

Reply:
> You do not include pyhf source code in the `MANIFEST.in`

Sorry, can you elaborate why not? [Pyramid](https://github.com/Pylons/pyramid/blob/30aae5f860e90126e45b45baeb6405568afd88b6/MANIFEST.in#L1) and [sphinx-py3doc-enhanced-theme](https://github.com/ionelmc/sphinx-py3doc-enhanced-theme/blob/50bd06442e9e2a5056ba4ab25e3cbb682f719e50/MANIFEST.in#L3) both are grafting `src` (though Ionel Mărieș [specifically argues for it](https://blog.ionelmc.ro/2014/05/25/python-packaging/))?

Reply:
Reading the docs: https://docs.python.org/2/distutils/sourcedist.html

I guess the phrasing is slightly ambiguous. If you want to include additional (non-default) files, then you use that.. but it looks like it's becoming a template to generate the MANIFEST so perhaps this is more correct.

Reply:
Yeah, as you said (from [the 3.7 docs](https://docs.python.org/3.7/distutils/sourcedist.html#specifying-the-files-to-distribute))

>  The typical way to do this is to write a manifest template, called `MANIFEST.in` by default. The manifest template is just a list of instructions for how to generate your manifest file, `MANIFEST`, which is the exact list of files to include in your source distribution. The **sdist** command processes this template and generates a manifest based on its instructions and what it finds in the filesystem.

so it seems that this would be an easy way to be quite sure that everything under `src/` gets packaged up by `sdist`.